lang/Spanish.ini

```diff
@@ -124,6 +124,7 @@ FORCE_4BY3 = "Forzar 4:3"
 PRELOAD_TEXTURES = "Precargar texturas"
 VSYNC = "Sincronización vertical"
 UNCAPPED_FRAMERATE = "FPS sin límite"
+REDUCE_FRAMERATE_FOCUS_LOSS = "Reducir la velocidad de fotogramas cuando la ventana no está enfocada"
```

Comment:
Did you test to see if this fits in the panel?

Reply:
Retranslated, new text fits in panel